docs/manuals/source/conf.py

```diff
@@ -302,51 +302,51 @@
-    r'http://www\.bareos\.org/.*': r'https://www\.bareos\.com/.*',
-    r'https://www\.bareos\.org/.*': r'https://www\.bareos\.com/.*',
-    r'https://www\.bareos\.com/whitepapers/.*': r'https://www\.bareos\.com/.*',
-    r'https://bugs\.bareos\.org/.*': r'https://bugs\.bareos\.org/my_view_page\.php',
-    r'https://download\.bareos\.org': r'https://download\.bareos\.org/bareos',
-    r'https://download\.bareos\.com': r'https://download\.bareos\.com/bareos',
+    r"https://bareos\.org/.*": r"https://bareos\.com/.*",
+    r"http://www\.bareos\.org": r"https://www\.bareos\.com/.*",
+    r"http://www\.bareos\.org/.*": r"https://www\.bareos\.com/.*",
+    r"https://www\.bareos\.org/.*": r"https://www\.bareos\.com/.*",
+    r"https://www\.bareos\.com/whitepapers/.*": r"https://www\.bareos\.com/.*",
+    r"https://bugs\.bareos\.org/.*": r"https://bugs\.bareos\.org/my_view_page\.php",
+    r"https://download\.bareos\.org": r"https://download\.bareos\.org/bareos",
+    r"https://download\.bareos\.com": r"https://download\.bareos\.com/bareos",
 }
 linkcheck_request_headers = {
-#    "https://www.sphinx-doc.org/": {
-#        "Accept": "text/html",
-#        "Accept-Encoding": "utf-8",
-#    },
+    #    "https://www.sphinx-doc.org/": {
+    #        "Accept": "text/html",
+    #        "Accept-Encoding": "utf-8",
+    #    },
     "*": {
         "Accept": "text/html,application/xhtml+xml",
     }
 }
-linkcheck_retries = 2   # default 1
+linkcheck_retries = 2  # default 1
 linkcheck_timeout = 10  # default 300
-linkcheck_workers = 2   # default 5
+linkcheck_workers = 2  # default 5
 linkcheck_anchors = True
 linkcheck_ignore = [
-    r'\.\./*',
-    r'http://localhost:*/',
-    r'https://\w+:\d+/bareos-webui/',
-    'https://UCS_SERVER/bareos-webui/',
-    'http://HOSTNAME/bareos-webui',
-    'http://localhost:9100',
-    'http://bareos:9100',
-    'http://bucket.s3_server/object',
-    'http://127.0.0.1:8000/docs',
-    'http://127.0.0.1:8000/redoc',
-    'https://pubs.vmware.com/vsphere-55/topic/com.vmware.vsphere.security.doc/*',
-    r'https://github\.com/bareos/bareos/pull/\d+', 
-    'https://www.glusterfs.org',                     #failing too often
-    'https://www.sphinx-doc.org/en/1.7/intro.html#',
+    r"\.\./*",
+    r"http://localhost:*/",
+    r"https://\w+:\d+/bareos-webui/",
+    "https://UCS_SERVER/bareos-webui/",
+    "http://HOSTNAME/bareos-webui",
+    "http://localhost:9100",
+    "http://bareos:9100",
+    "http://bucket.s3_server/object",
+    "http://127.0.0.1:8000/docs",
+    "http://127.0.0.1:8000/redoc",
+    "https://pubs.vmware.com/vsphere-55/topic/com.vmware.vsphere.security.doc/*",
+    r"https://github\.com/bareos/bareos/pull/\d+",
+    "https://www.glusterfs.org",  # failing too often
+    "https://www.sphinx-doc.org/en/1.7/intro.html#",
 ]
-#linkcheck_auth = [
+# linkcheck_auth = [
 #  ('https://foo\.yourcompany\.com/.+', ('johndoe', 'secret')),
 #  ('https://.+\.yourcompany\.com/.+', HTTPDigestAuth(...)),
```

Comment:
```suggestion
#   ('https://foo\.yourcompany\.com/.+', ('johndoe', 'secret')),
#   ('https://.+\.yourcompany\.com/.+', HTTPDigestAuth(...)),
```
Otherwise removing the comment+space columns will broke indentation

Reply:
Done by `bareos-check-sources`.

Reply:
ok